test/jdk/javax/swing/text/html/FrameView/4463014/frameset.html

```diff
@@ -0,0 +1,11 @@
+<HTML>
+ <HEAD>
+   <TITLE>
+     Manual test for bug 4463014
+   </TITLE>
+ </HEAD>
+ <FRAMESET COLS="150,*">
+  <FRAME SRC="frame1.html">
+  <FRAME name="main" SRC="frame2.html">
+ </FRAMESET>
+</HTML>
```

Comment:
A blank line in the end of file?

Reply:
Not that I can see .. even doing a binary dump, and I didn't edit this file at all, I just moved it from closed to open.